post-scan-actions/aws-python-promote-or-quarantine/template.yml

```
@@ -76,8 +98,71 @@ Conditions:
     !Not [!Equals [!Ref QuarantineBucketName, '']]
   PutACL:
     !Not [!Equals [!Ref ACL, '']]
+  IsBucketSSEKMSEnabled:
+    !Not [!Equals [!Ref KMSKeyARN, '']]
+  HasPermissionsBoundary:
+    !Not [!Equals [!Ref PermissionsBoundary, '']]
+  VpcEnabled: !Not [!Or [!Equals [!Ref VpcSubnetId, ''], !Equals [!Ref VpcSecurityGroupId, '']]]
 
 Resources:
+  LambdaExecutionRole:
```

In case we'll have another lambda in the stack
```suggestion
  PromoteOrQuarantineLambdaExecutionRole:
```